File: app/src/main/java/com/autowheel/bangbang/MyApplication.java
package com.autowheel.bangbang;

import android.app.Application;

import com.didichuxing.doraemonkit.DoraemonKit;

/**
 * Created by Xily on 2020/3/5.
 */
public class MyApplication extends Application {
    private static MyApplication instance;
    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
        DoraemonKit.install(this);
    }

    public static MyApplication getInstance() {
        return instance;
    }
}

File: app/src/main/java/com/autowheel/bangbang/MyApplication.kt
package com.autowheel.bangbang

import android.app.Application
import com.didichuxing.doraemonkit.DoraemonKit
import com.vanniktech.emoji.EmojiManager
import com.vanniktech.emoji.ios.IosEmojiProvider
import org.litepal.LitePal

/**
 * Created by Xily on 2020/4/6.
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        instance = this
        LitePal.initialize(this)
        DoraemonKit.install(this)
        EmojiManager.install(IosEmojiProvider())
    }

    companion object {
        private lateinit var instance: MyApplication
        @JvmStatic
        fun getInstance(): MyApplication {
            return instance
        }
    }
}